torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -256,6 +274,57 @@ RegisterOperators reg({
             return 0;
           };
         }),
+    Operator(
+        "aten::device(str a) -> Device",
+        [](const Node* node) -> Operation {
+          return [](Stack& stack) {
+            push(stack, c10::Device(pop(stack).toStringRef()));
```

Comment:
Why `toStringRef`? `IValue` supports `Device`

Reply:
This shouldn't have been added - it already exists. Will delete. Good catch